web/pages/_app.tsx

```diff
@@ -76,8 +77,10 @@ function MyApp({ Component, pageProps }: AppProps<ManifoldPageProps>) {
       <AuthProvider serverUser={pageProps.auth}>
         <NativeMessageListener />
         <QueryClientProvider client={queryClient}>
-          <Welcome />
-          <Component {...pageProps} />
+          <SearchProvider>
```

Comment:
I don't see why this is a context instead of just a component? Nothing ever uses the context.

Reply:
the button in the sidebar uses it to open the modal.

I suppose it could be a component in the sidebar, and if we want it to share state across pages later (like for search history or cached results) we could use local storage or turn sidebar into a next 13 layout